### PR TITLE
Second branch

### PR DESCRIPTION
**Pull request by MagdaMatetovici**
Updated the package. Added some documentation and tests. And added functionality to choose a location from a map.

### CODE REVIEW

File: ShinyWheater/tests/testthat/test-Main_Shiny.R
Comment (HABT29) [inline ShinyWheater/tests/testthat/test-Main_Shiny.R:1]: You could delete this right?

**Review by HABT29 — COMMENTED** [review]
Overall I think you going in a very good direction, a lot looks already quite advanced. Regarding the testing, I think you came up with nice tests that actually tell you something about the functionality of your functions. For the documentation, you use a lot of comments which is nice, and your documentation of the function to run your app also looks good. Maybe you could add a bit of documentation for yourself, instead or complementing your comments.

**Comment by MarliesRoozendaal** [comment]
You're going fast! I think its great that you commit every change, so you can trace back your changes easily. I agree with the feedback on your tests and documentation, good job!

File: ShinyWheater/tests/testthat/test-Main_Shiny.R
Comment (MagdaMatetovici) [reply ShinyWheater/tests/testthat/test-Main_Shiny.R:1]: I'm not sure, I was thinking to see if I can make a test for the whole shiny app but I let this for later haha, after I finish all of it. But if I do not write a test for it, I will delete it!

**Pull request merged**